packages/builders/scripts/orchestration/init-stakeBld.js

```diff
@@ -13,12 +27,36 @@ export const defaultProposalBuilder = async ({ publishRef, install }) =>
             install('@agoric/orchestration/src/examples/stake-bld.contract.js'),
           ),
         },
+        options,
```

Comment:
are the options the same on all these example contracts? if so, I wonder about a common builder with an arg to pick which core eval to build

Reply:
More or less, yes. I didn't go with a common builder (yet) but added a `parseChainHubOpts` helper